examples.js

```diff
@@ -0,0 +1,107 @@
+function examples() {
+
+  setTimeout(blastOff, 86400000);
+
+
+  const locations = ["Austin", "New York", "San Francisco"];
+  locations.forEach(l => {
+    doStuff();
+    doSomeOtherStuff();
+    dispatch(l);
+  });
+
+
+  function createMenu(title, body, buttonText, cancellable) {
+    // ...
+  }
+  
+  createMenu("Foo", "Bar", "Baz", true);
+
+
+  function emailClients(clients) {
+    clients.forEach(client => {
+      const clientRecord = database.lookup(client);
+      if (clientRecord.isActive()) {
+        email(client);
+      }
+    });
+  }
+
+
+  const date = new Date();
+  addToDate(date, 1);
```

Comment:
¿Qué es 1? Podrías hacer la función más descriptiva.
```suggestion
  const date = new Date();
  addMonthToDate(1, date);
```